data/widgets/card/list.ui

```diff
@@ -92,6 +92,18 @@
                 </style>
               </object>
             </child>
+            <child>
+              <object class="GtkFrame" id="context-frame">
```

Comment:
Any particular reason why to use a frame and not a box?
theses days all widgets can be styled, also this widget should be no-show-all